src/server/AccountsServer.spec.js

```diff
@@ -23,4 +26,72 @@ describe('Accounts', () => {
       expect(Accounts.instance.db).to.equal(db);
     });
   });
+  describe('createUser', () => {
+    const db = {
+      findUserByUsername: () => Promise.resolve(),
+      findUserByEmail: () => Promise.resolve(),
+      createUser: () => Promise.resolve(),
+    };
+    beforeEach(() => {
+      Accounts.config({}, db);
+    });
+    it('requires username or an email', async () => {
+      try {
+        await Accounts.createUser({
+          password: '123456',
+          username: '',
+          email: '',
+        });
+        expect.fail();
+      } catch (err) {
+        const { message } = err.serialize();
+        expect(message).to.eql('Username or Email is required');
+      }
+    });
+    it('throws error if username exists', async () => {
+      Accounts.config({}, {
+        ...db,
+        findUserByUsername: () => Promise.resolve('user'),
+      });
+      try {
+        await Accounts.createUser({
+          password: '123456',
+          username: 'user1',
+          email: '',
```

Comment:
Maybe add a test case where email or username is undefined? This should still work as long as one of them is a valid filed.